### ISSUE TITLE
Add SSL via let's encrypt

### ISSUE THREAD

**Issue by issmirnov** [debops/ansible-dokuwiki#3]
Thank you for making a kickass playbook. If you get a chance, could you add a step to encrypt the wiki with SSL? I see there's some [ansible roles](https://github.com/PeterMosmans/ansible-role-letsencrypt) already created that you could possibly hook into.

Thanks!

**Comment by drybjed** [comment]
Support for SSL, and Let's Encrypt is already there, but perhaps not obvious. :-)

This role is part of the [DebOps Project](http://debops.org/). The DokuWiki (application) can use [debops.nginx](https://github.com/debops/ansible-nginx) role to manage its `nginx` webserver. The `debops.nginx` role supports the environment managed by [debops.pki](https://github.com/debops/ansible-pki) Ansible role and can aid it in ACME/Let's Encrypt activation. The `debops.pki` role manages SSL certificates and allows you to either create a private, internal CA, use your existing external CA via scripting, or request an ACME certificate from Let's Encrypt.

In other words, I think that you found the DokuWiki role and installed it "directly" on a host? You could try setting up a new DebOps-managed host using the main playbook. After installing `nginx` on it, if the conditions are right (public IP address, properly configured DNS), DebOps will automatically request ACME certificates and switch to them. After that, your DokuWiki installation should have properly configured HTTPS "out of the box".
